.travis.yml

```diff
@@ -14,24 +14,31 @@ cache:
   timeout: 600
   directories:
     - $HOME/cache
-
 jobs:
   include:
     - stage: build third party
       script:
         - cd earth_enterprise/src
-        - if [ -f $HOME/cache/third_party.tgz ]; then tar xf $HOME/cache/third_party.tgz; fi
-        - scons -j3 release=1 third_party > build.log
+        - if [ -f $HOME/cache/third_party$CPP_STD.tgz ]; then tar xf $HOME/cache/third_party$CPP_STD.tgz; fi
+        - scons -j3 release=1 cpp_standard=gnu++$CPP_STD third_party > build.log
         - mkdir -p $HOME/cache
-        - tar cfz $HOME/cache/third_party.tgz NATIVE-REL-x86_64 .sconsign.dblite .sconf_temp
+        - tar cfz $HOME/cache/third_party$CPP_STD.tgz NATIVE-REL-x86_64 .sconsign.dblite .sconf_temp
+      env:
```

Comment:
This is the problem, anything in a list has to have `- ` in front of it. Otherwise, whatever is last simply replaces what was there before.

Reply:
Try updating it to exactly what I posted: 

```
       - env:
         CPP_STD=98
       - env:
         CPP_STD=11
```

Reply:
I personally like the simplicity of JSON.  XML use to be my favorite generic and loosely coupled data exchange format but when I started using JSON I switched to liking JSON better.  Yea I'm not a fan at all of yaml...